transformations/color_transformation/transformation.py

```diff
@@ -0,0 +1,95 @@
+import json
+import os
+import random
+
+import nltk
+from nltk.tokenize.treebank import TreebankWordDetokenizer
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+def get_(big_list, small_list):
+    big_str = " ".join(big_list)
+    small_str = " ".join(small_list)
+
+    big_str.index(small_str)
+
+
+"""
+Base Class for implementing the different input transformations a generation should be robust against.
+"""
+
+
+class ColorTransformation(SentenceOperation):
+    tasks = [
+        TaskType.TEXT_CLASSIFICATION,
+        TaskType.TEXT_TO_TEXT_GENERATION,
+        TaskType.TEXT_TAGGING,
+    ]
+    languages = ["en"]
+
```

Comment:
Can you add the missing [keywords](https://github.com/GEM-benchmark/NL-Augmenter/blob/main/docs/keywords.md) variable here.

Reply:
Done! Thanks for the pointer.